core/src/main/scala/com/salesforce/op/dsl/RichDateFeature.scala

```diff
@@ -85,17 +91,22 @@ trait RichDateFeature {
      * @param dateListPivot name of the pivot type from [[DateListPivot]] enum
      * @param referenceDate reference date to compare against when [[DateListPivot]] is [[SinceFirst]] or [[SinceLast]]
      * @param trackNulls    option to keep track of values that were missing
+     * @param circularDateRepresentations list of all the circular date representations that should be included
+     *                                    feature vector
      * @return result feature of type Vector
      */
     def vectorize
     (
       dateListPivot: DateListPivot,
       referenceDate: JDateTime = TransmogrifierDefaults.ReferenceDate,
       trackNulls: Boolean = TransmogrifierDefaults.TrackNulls,
+      circularDateRepresentations: Seq[TimePeriod] = TransmogrifierDefaults.CircularDateRepresentations,
```

Comment:
this is a ridiculously long param name ;) can we perhaps shorten to `circularDateRep`? (same elsewhere)